<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>560</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_tan">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>175</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>tan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 224, 142);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>175</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 219, 128);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777212</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 215, 218);
	font: 16pt &quot;.AppleSystemUIFont&quot;;
	border: 1px solid gray;
	border-radius: 20px;

	</string>
    </property>
    <property name="text">
     <string>X max</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777212</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(217, 215, 218);
	font: 16pt &quot;.AppleSystemUIFont&quot;;
	border: 1px solid gray;
	border-radius: 20px;

	</string>
    </property>
    <property name="text">
     <string>Y max</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSpinBox_Ymax">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>443</y>
      <width>71</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_div">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background-color: rgb(195, 209, 77);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mod">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>mod</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="result">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>49</y>
      <width>451</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"> qproperty-alignment: 'AlignVCenter | AlignLeft';
 qproperty-wordWrap: true;
background-color: rgb(217, 215, 218);

</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_del">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>109</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background-color: rgb(195, 209, 77);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>&lt;&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AC">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>109</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background-color: rgb(195, 209, 77);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_log">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ln">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>ln</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_plus">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background-color: rgb(195, 209, 77);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSpinBox_X">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>112</y>
      <width>121</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;.AppleSystemUIFont&quot;
</string>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 219, 128);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_RBracket">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 134, 67);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pow">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background-color: rgb(195, 209, 77);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_acos">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>acos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>175</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>sin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mult">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background-color: rgb(195, 209, 77);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>175</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 219, 128);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_X">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>109</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
font: 16pt &quot;.AppleSystemUIFont&quot;;
	border-radius: 20px;
background-color:  rgb(217, 215, 218);</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_sqrt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>sqrt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="error">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>3</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(237, 214, 76);
</string>
    </property>
    <property name="text">
     <string>All Right</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_min">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>175</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background-color: rgb(195, 209, 77);
	font: 16pt &quot;.AppleSystemUIFont&quot;
</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_equal">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>109</y>
      <width>127</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background-color: rgb(195, 209, 77);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cos">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>175</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>cos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>175</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 219, 128);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 219, 128);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 219, 128);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 219, 128);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_LBracket">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 134, 67);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_atan">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>atan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_graph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;
background-color: rgb(217, 215, 218);
	border: 1px solid gray;
	border-radius: 20px;</string>
    </property>
    <property name="text">
     <string>график</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_point">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>370</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
background-color: rgb(199, 134, 67);
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>305</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 219, 128);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border: 1px solid gray;
	border-radius: 20px;
background:rgb(237, 219, 128);
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_asin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 187, 116);
	border: 1px solid gray;
	border-radius: 20px;
	font: 16pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="text">
     <string>asin</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="DSpinBox_Xmax">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>443</y>
      <width>71</width>
      <height>55</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;.AppleSystemUIFont&quot;</string>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
